src/components/Discover/index.tsx

```diff
@@ -58,52 +58,56 @@ const Discover = () => {
   return (
     <>
       <PageTitle title={intl.formatMessage(messages.discover)} />
-      {hasPermission([Permission.MANAGE_REQUESTS, Permission.RECENT_VIEW], {
-        type: 'or',
-      }) && (
+      {(!media || !!media.results.length) &&
+        !mediaError &&
```

Comment:
I haven't tested this locally yet, but I imagine we will get pop-in for these sliders since we are not rendering them at all until the media comes back from the request. The alternative is having them... flash away. I don't think either solution will look very good 🤔

Reply:
Actually, it's the same logic as for the watchlist slider -- it flashes away . But I think that's more okay, since it is less likely for these sliders to actually be empty?

Reply:
That's true that the watchlist flashes away as well. I don't like how that one behaves either, haha. Not sure if there is a great compromise here 🤔